plotly/tools.py

```diff
@@ -99,7 +103,8 @@ def ensure_local_plotly_files():
 
 ### credentials tools ###
 
-def set_credentials_file(username=None, api_key=None, stream_ids=None):
+def set_credentials_file(username=None, api_key=None, proxy_username=None,
+                         proxy_passwd=None, stream_ids=None):
```

Comment:
move `proxy_username` and `proxy_password` to the very end for backwards compatibility - some folks might be setting up their creds as positional arguments